sw/device/lib/testing/i2c_testutils.h

```diff
@@ -8,6 +8,7 @@
 #include <stdint.h>
 
 #include "sw/device/lib/dif/dif_i2c.h"
+#include "sw/device/lib/runtime/ibex.h"
```

Comment:
I think this include should actually be moved into `sw/device/tests/sim_dv/i2c_device_tx_rx_test.c` because the macro isn't used inside `i2c_testutils.c` right?